bin/reth/src/cli/mod.rs

```diff
@@ -18,6 +18,9 @@ use reth_cli_commands::db;
 use reth_cli_runner::CliRunner;
 use reth_db::DatabaseEnv;
 use reth_node_builder::{NodeBuilder, WithLaunchContext};
+use reth_optimism_cli::commands::{
+    import::ImportOpCommand, import_receipts::ImportReceiptsOpCommand,
+};
```

Comment:
please remove these imports and use full paths instead because this is feature gated